packages/components/socialFeed/NewsFeed/MapModal/index.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { Suspense, lazy, useEffect, useState } from "react";
```

Comment:
Nope it's not done https://github.com/TERITORI/teritori-dapp/pull/898/files#r1481219515
It is too long again, and the `index.tsx` means nothing for this component, so please change the name of that (and don't forget to split it)

Reply:
Ok, done